Recrutify/Recrutify.Services/DTOs/ProjectDTO.cs

```diff
@@ -1,18 +1,19 @@
 ﻿using System;
 using System.Collections.Generic;
+using Recrutify.DataAccess.Models;
 
 namespace Recrutify.Services.DTOs
 {
     public class ProjectDTO : BaseProjectDTO
     {
         public Guid Id { get; set; }
 
-        public IEnumerable<StaffDTO> Managers { get; set; }
+        public IEnumerable<Staff> Managers { get; set; }
```

Comment:
why Staff and not StaffDTO, we don't use models in dtos

Reply:
done